src/contracts/proxies/HaiSafeManager.sol

```diff
@@ -23,6 +24,9 @@ contract HaiSafeManager {
   // --- Registry ---
   address public safeEngine;
 
+  // --- ERC721 ---
+  address public vault721;
```

Comment:
Should the DAO have a setter for this?

Reply:
do we foresee upgrading/migrating Vault721?